lib/form.js

```diff
@@ -112,9 +113,16 @@ Form.prototype.is_email = function (email) {
   return regexp.test(email)
 }
 
+Form.prototype.set_mk_email = function (mk_email) {
+  var domain = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@$/g
```

Comment:
@rafikahMH  is this the domain or is this the local part of the email (everything before the @)

Reply:
my bad, this should have been labelled local_part instead of domain